src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -456,6 +471,10 @@ def _do_upgrade(self):
             ):
                 return
 
+            if need_upgrade:
+                self.upgrade_state.info_str = 'Currently upgrading %s daemons' % (daemon_type)
```

Comment:
```suggestion
                self.upgrade_info_str = 'Currently upgrading %s daemons' % (daemon_type)
```

I'd avoid storing this persistently.